config/config.go

```diff
@@ -150,5 +144,14 @@ func GetOptionalParameters(config *Config, defaultLidarDataRateMsec, defaultIMUD
 		mapRateSec = *config.MapRateSec
 	}
 
-	return lidarDataRateMsec, imuName, imuDataRateMsec, mapRateSec, nil
+	enableMapping := false
+	if config.CloudStoryEnabled {
+		if config.EnableMapping == nil {
+			logger.Debugf("no enable_mapping given, setting to default value of false", defaultMapRateSec)
+		} else {
+			enableMapping = config.CloudStoryEnabled
+		}
+	}
```

Comment:
thoughts on a warning if `cloud_story_enabled = true` and `enable_mapping = true`?